packages/cli/src/License.ts

```diff
@@ -64,6 +67,7 @@ export class License {
 				loadCertStr: async () => this.loadCertStr(),
 				saveCertStr,
 				deviceFingerprint: () => instanceId,
+				onFeatureChange,
```

Comment:
didn't we want to set `offlineMode` to true on workers?

Reply:
`const offlineMode = !isMainInstance;`

Reply:
ah, sorry, I didn't realize that GitHub was hiding these lines 🤦‍♂️